Teradata/covalent
# Feat/vui 913 copy code button

## Description


### What's included?

- Copy button for code snippet

#### Test Steps

- [ ] `npm run serve`
- [ ] then check flavored markdown examples:
http://localhost:4200/#/components/flavored-markdown/examples
- [ ] check the markdown navigator example - http://localhost:4200/#/components/markdown-navigator/examples
- check the service example and click on the open button
- a new dialog will be opened, click on the 3rd link `browser-support`.
- click on the copy button and paste it.
- it should able to copy the code and paste it.


#### General Tests for Every PR


##### Screenshots or link to StackBlitz/Plunker
![image](https://user-images.githubusercontent.com/12620590/89579795-3da81980-d7e9-11ea-849e-7fccf969f629.png)
